### DISCUSSION TITLE
I can no longer see my docker images

### DISCUSSION THREAD

**Discussion by Trolldemorted** [community/community#26596] [Actions]
<p>I have uploaded a docker image a few weeks ago and set it public. Anyone can download it with <code>docker pull ghcr.io/trolldemorted/bsprak:latest</code>, so far so good.</p>
<p>Today I noticed I can no longer see my own image:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/0/30417b59e4939833b6ef388eacbe56ed2845a715.png" data-download-href="/uploads/short-url/6STdbURf7ARHI8h3wZFfhAnKpXD.png?dl=1" title="grafik" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10261186/181098608-1155745f-6591-4a54-aff5-696485580e1e.png" alt="grafik" data-base62-sha1="6STdbURf7ARHI8h3wZFfhAnKpXD" width="689" height="421" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30417b59e4939833b6ef388eacbe56ed2845a715_2_689x421.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30417b59e4939833b6ef388eacbe56ed2845a715_2_1033x631.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/0/30417b59e4939833b6ef388eacbe56ed2845a715.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30417b59e4939833b6ef388eacbe56ed2845a715_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">grafik</span><span class="informations">1283×784 48.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So I am unable to to things like see the size, set it to private, etc… I assume this is not intended?</p>

**Comment by whitneyimura** [comment, marked as answer]
<p>Hey there! Would you mind trying again and seeing whether you can see your packages now?</p>

**Comment by Trolldemorted** [comment]
<p>Hey, yeah now I can see it again!</p>

**Comment by king40or1** [comment]
<p>Hey what did you do to fix this?</p>
